app/_config/email.yml

```diff
@@ -1,9 +1,8 @@
-# See https://docs.silverstripe.org/en/4/developer_guides/email/ for additional information about setting up email
 ---
 Name: project-emailconfig
 After:
   - '#emailconfig'
 ---
-SilverStripe\Core\Injector\Injector:
-  Swift_Transport:
-    class: Swift_SendmailTransport
+SilverStripe\Control\Email\Email:
+  admin_email:
+    admin@demo.silverstripe.org: 'Admin-email'
```

Comment:
Why are you adding an email address here?

Reply:
In case any emails do get sent out (erronously?) they'll be easier to see where they come from

Reply:
Deleted